src/main/kotlin/cn/yiiguxing/plugin/translate/trans/deepl/DeeplTranslator.kt

```diff
@@ -109,27 +109,37 @@ object DeeplTranslator : AbstractTranslator(), DocumentationTranslator {
     }
 
     override fun doTranslate(text: String, srcLang: Lang, targetLang: Lang): Translation {
-        return SimpleTranslateClient(this, DeeplTranslator::call, DeeplTranslator::parseTranslation).execute(
-            text,
-            srcLang,
-            targetLang
-        )
+        return SimpleTranslateClient(
+            this,
+            { _, _, _ ->call(text, srcLang, targetLang, false) },
+            DeeplTranslator::parseTranslation
+        ).execute(text, srcLang, targetLang)
     }
 
-    private fun call(text: String, srcLang: Lang, targetLang: Lang): String {
+    private fun call(text: String, srcLang: Lang, targetLang: Lang, isDocument: Boolean): String {
         val settings = Settings.deeplTranslateSettings
         val privateKey = settings.getAppKey()
         val splitPrivatekeys = privateKey.split(":")
         val isFree = (splitPrivatekeys.size == 2 && splitPrivatekeys[1] == "fx")
```

Comment:
Using `val isFree = privateKey.endsWith(":fx")` here will be more concise and efficient.

Reply:
Thanks. Corrected.